src/reverse_complement.rs

```diff
@@ -10,12 +10,11 @@ extern crate rayon;
 extern crate memchr;
 
 use std::io::{Read, Write};
-use std::{io, ptr, slice};
+use std::{cmp, io, mem};
 use std::fs::File;
 
 struct Tables {
```

Comment:
I used the `fn build_table() -> [u8; 256]` approach, because I found it the most readable.